data_module.py

```diff
@@ -12,6 +15,7 @@ def __init__(self,
                 test_datasets_names: list,
```

Comment:
You are assuming a specific naming structure, please document how you expect the naming to look like.